notebooks/06-cifar10-baseline.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>I think it's <code>Bolts</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/pytorch-lightning/pull/4818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>This was taken from https://github.com/PyTorchLightning/pytorch-lightning/pull/4555</p>